docs/naps/0-nap-process.md

```diff
@@ -0,0 +1,298 @@
+---
+Author: "Juan Nunez-Iglesias <mailto:jni@fastmail.com>"
+Author: "Andy Sweet <mailto:andrewdsweet@gmail.com>"
+Author: "Kevin Yamauchi <mailto:kevin.yamauchi@gmail.com>"
+Created: '2022-03-23'
+Status: Active
+Type: Process
+---
+
+(nap0)=
+
+# NAP 0 — Purpose and Process
+
+## What is a NAP?
+
+NAP stands for Napari Advancement Proposal. A NAP is a design document
+providing information to the community, or describing a new feature for
+napari or its processes or environment. The NAP should provide a rationale
```

Comment:
This portion needs to be more explicit about the scope of NAPs & ensure that the scope is consistent with other governance documents, such as https://napari.org/community/governance.html

1. Is the NAP process limited in scope to napari, as is described here?
2. Does it extend to other official napari projects that live under the `napari` Github organization, such as magicgui, napari-sphinx-theme, cookiecutter-napari-plugin, and npe2?
3. Does it extend to pieces of the napari ecosystem that are beyond napari's governance, such as plugins (including napari-error-reporter), the napari hub, Bioimage Model Zoo, PyPI, Github, etc?

(2) seems like the scope which is most consistent with the scope of the existing governance documents: https://napari.org/community/governance.html